src/modules/Shows.js

```diff
@@ -0,0 +1,36 @@
+class Shows {
+    static apiEndPoint = "https://api.tvmaze.com/shows/1/episodes";
+    static getShows = async () => {
+      const response = await fetch(this.apiEndPoint).then((response) =>
+        response.json()
+      );
+      return response;
+    };
+  
+    static updateUI = async () => {
+      const shows = await Shows.getShows();
+      const card = document.getElementById("card");
+      card.classList.add("wrap", "flex");
```

Comment:
Please do not add unused class names (`wrap` in this case) unless you are going to use them to style an element in your CSS, as this will make your code unpleasant and ambiguous. 😕